src/mobu/handlers/external.py

```diff
@@ -232,3 +253,33 @@ async def get_summary(
     context: Annotated[RequestContext, Depends(context_dependency)],
 ) -> list[FlockSummary]:
     return context.manager.summarize_flocks()
+
+
+@external_router.post(
+    "/github/webhook",
+    summary="GitHub webhooks",
+    description="This endpoint receives webhook events from GitHub.",
+    status_code=202,
+)
```

Comment:
I would move this into a separate `github.py` file and a separate router just for readers of the code, since the caller of this API is so different.